### DISCUSSION TITLE
change Winddirection graph to be more indicative of the direction

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#20540] [Dashboard]
currently wind direction shows like

<img width="588" alt="Scherm­afbeelding 2024-04-17 om 14 13 39" src="https://github.com/home-assistant/frontend/assets/33354141/70c460a0-e10a-4f42-b6a0-78d54465e20d">

in a core entities card, clicking on the entity's more-info

which is because it treats the state as numeric (which it is, so not complaining about an issue here..)

it however does not indicate the direction history to the viewer very understandably though

This FR is hoping something like this ([history-explorer-card](https://github.com/alexarch21/history-explorer-card), no longer maintained) could be done

![image](https://github.com/home-assistant/frontend/assets/33354141/fa2cd41a-3682-4bc4-b507-48f5c776052b)

or maybe even something like custom cards like [Windrose card](https://github.com/aukedejong/lovelace-windrose-card)
![image](https://github.com/home-assistant/frontend/assets/33354141/b01b76f6-6883-47ed-9028-95885a067852)

or [compass card](https://github.com/tomvanswam/compass-card):

![image](https://github.com/home-assistant/frontend/assets/33354141/724926e9-f984-4688-9a18-1c1f79d344c7)

thanks for considering

**Comment by dsellers1** [comment]
I wonder if this could be incorporated as a Show As. All it would need to do would be to take the bearing, divide by 22.5, and return the direction.

I helped someone on Discord incorporate something similar into their frontend.
```yaml
type: custom:button-card
name: |
  [[[
    let bearing = 359
    const directions = ["N","NNE","NE","ENE","E","ESE","SE",
        "SSE","S","SSW","SW","WSW","W","WNW","NW","NNW"]
    return directions[parseInt( bearing / 22.5)]
  ]]]
  ```
  *(Note: the hard-coded bearing variable was used for testing purposes.)*
![image](https://github.com/home-assistant/frontend/assets/67642332/53467eb5-f9f2-48c6-81db-028c4fe5d436)

**Comment by Mariusthvdb** [comment]
yes, as mentioned in Discord, we can write up al sorts of templates and button-cards. But my main ask here would be to prevent core more-info from displaying a 'known' entity's unit of measurement as another, and as a result of that render a more-info which obviously isn't designed for  that particular entity.

Dont think the show-as currently takes that into account (mainly focusses on the icon), or even the unit option we now get for windspeed. 

Agree it would be nice if we could configure some setting in the options to fix it, great idea
